app/assets/stylesheets/application.css.scss

```diff
@@ -98,7 +96,7 @@ a:hover {
   text-align: center;
 }
 
-.flash_notice {
+.flash_notice, .flash_error {
```

Comment:
Selector `flash_notice` should be written in lowercase with hyphens<br>Selector `flash_error` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line